src/Kuzzle.js

```diff
@@ -310,6 +296,28 @@ function Kuzzle (host, options, cb) {
   }
 }
 
+Kuzzle.prototype = new KuzzleEventEmitter();
+
+/**
+* Emit an event to all registered listeners
+* An event cannot be emitted multiple times before a timeout has been reached.
+*/
+Kuzzle.prototype.emit = function(eventName) {
+  var
+    now = Date.now(),
+    protectedEvent = this.protectedEvents && this.protectedEvents[eventName];
```

Comment:
nitpicking: it's unnecessary to verify the existence of `this.protectedEvents` as we're guaranteed to have this property set in any `Kuzzle` instance.